index.d.ts

```diff
@@ -1,3 +1,21 @@
+export interface Options {
+	/**
+	Wether it should automatically prepend `https://` in case the input has no protocol defined.
```

Comment:
You are describing the implementation detail, not what problem it solves for the user.

Reply:
Gotcha. I've updated the description. Let me know if it can be further improved.